core/package.json

```diff
@@ -19,6 +19,7 @@
     "lodash.template": "^4.5.0",
     "mock-local-storage": "^1.1.23",
     "node-fetch": "2.6.7",
+    "rollup": "^3.29.5",
```

Comment:
I think we can remove this rollup line and skip core update altogether. 

I would have to guess that, the Dependabot alert is **just outdated** for core regarding this rollup issue.

The warning most likely arised because down below here we have
  `"vite": "^4.0.0"` 
which at the time the warning was produced installed latest v4 vite = **4.5.3** due to the **^** sign

Currently `"vite": "^4.0.0"`  will install v4 vite = **4.5.5** as it was released recently as a patch to this issue. 


So the only thing to do here would be to have a **clean install** of depdencies and push the resulting **package-lock.json** 🤔

Reply:
To reproduce/test this you may try 
1. Fix vite in this file to   `"vite": "4.5.3"`
2. run **npm install**  (I usually delete node_modules and pacakge-lock before to make sure no cache is reused)
3. Check the output how many warnings it shows. (You may use **npm audit** to find the Dom Clobbering one)
4. Now change to `"vite": "^4.0.0"` and run a **clean npm install** again. You can see the dom clobering one is gone.